randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -1299,6 +1305,10 @@ Extra - map_name: rm_a3b03
 > Dock to Breeding Grounds Crossroads; Heals? False
   * Layers: default
   * Open Passage to Breeding Grounds Crossroads/Dock to Breeding Grounds Gamma Nest West
+  > Dock to Outer Caverns Crossroads
+      Any of the following:
+          Space Jump or Can Use Spider Ball
+          Hi-Jump Boots and Walljump (Intermediate)
```

Comment:
you can reach that with high jump? afaik its  just barely out of reach

Reply:
It is possible. It is one tile though so this should maybe be bumped to Advanced WJ

Reply:
yeah bump to advanced then